pom.xml

```diff
@@ -882,6 +882,11 @@
                 <artifactId>gt-xsd-wps</artifactId>
                 <version>${version.geotools}</version>
             </dependency>
+            <dependency>
+                <groupId>org.hibernate</groupId>
+                <artifactId>hibernate-annotations</artifactId>
+                <version>3.4.0.GA</version>
+            </dependency>
```

Comment:
When building from a clean maven repository, the build failed because it could not download `javax.transaction:jta:1.0.1B` which is a transitive dependency of hibernate. Checking maven central, it appears that this artifact no longer exists because it is a pom type artifact instead of a jar artifact.

Hibernate is a transitive dependency of `org.picketbox:picketbox-infinispan` which is used in the `web-services/security` maven module. This version of `hibernate-annotations`  depends on a different version of jta, where a jar does exist.